<launch>
  <arg name="scan_phase"  default="0.0"/>
  <arg name="model" default="Pandar40P"/>
  <arg name="device_ip" default="192.168.1.201"/>
  <arg name="calibration" default="$(find-pkg-share pandar_pointcloud)/config/40p.csv"/>

  <!-- pandar cloud -->
  <node pkg="pandar_pointcloud" exec="pandar_cloud_node" name="pandar_cloud" output="screen" >
    <remap from="pandar_points" to="pointcloud_raw" />
    <remap from="pandar_points_ex" to="pointcloud_raw_ex" />
    <param name="scan_phase"  type="double" value="$(var scan_phase)"/>
    <param name="model"  type="string" value="$(var model)"/>
    <param name="device_ip" type="string" value="$(var device_ip)"/>
    <param name="calibration" type="string" value="$(var calibration)"/>
  </node>
</launch>
